cmd/minikube/cmd/start_flags.go

```diff
@@ -98,6 +99,7 @@ const (
 	nodes                   = "nodes"
 	preload                 = "preload"
 	deleteOnFailure         = "delete-on-failure"
+	dockerBaseImage         = "docker-base-image"
```

Comment:
I would change this to base-image, since this is not docker specific, and could be podman too